src/components/SignerPanel/ActionPathsContent.js

```diff
@@ -75,18 +75,18 @@ class ActionPathsContent extends React.Component {
                 }
                 if (type === 'app') {
                   return (
-                    <ButtonText
+                    <Link
                       key={index}
                       href={`#${getAppPath({
                         dao,
                         instanceId: 'permissions',
                         params: `app.${value.proxyAddress}`,
                       })}`}
-                      target="_blank"
+                      focusRingSpacing={[2, 2]}
```

Comment:
Is this needed? I was more seeing it as an internal prop that `Link` uses, but if we need to use it, maybe we should make it part of `Link` prop types and document it.

Reply:
Admittedly this is a bit of a quickfix, but this was the reason:

Without the `focusRingSpacing` (using defaults):

<img width="565" alt="Screen Shot 2019-08-30 at 9 30 08 AM" src="https://user-images.githubusercontent.com/4166642/64001746-c9f58b00-cb08-11e9-8bb5-cb81520875a6.png">

With the override:
<img width="494" alt="Screen Shot 2019-08-30 at 9 27 36 AM" src="https://user-images.githubusercontent.com/4166642/64001742-c7933100-cb08-11e9-8e4c-509b4a5f2c0c.png">

Reply:
mmm yes maybe we should fix the default then. I agree the first one could be improved, but the second one is too close to the characters IMO.

It might be a good case for `em`, so that it would depend on the text size… but the problem is that I don’t think there is a way to stop depending on the text size after a certain point (we don’t want huge spacing on large titles). But we could do some tests at different text sizes, maybe using a combination of `px` and `em` using `calc()`, and see if it works.